Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -542,6 +542,30 @@ class PaymentSheetStandardLPMUITests: PaymentSheetUITestCase {
         payButton.tap()
     }
 
+    func testAmazonPayPaymentMethod() throws {
+        var settings = PaymentSheetTestPlaygroundSettings.defaultValues()
+        settings.customerMode = .new
+        settings.apmsEnabled = .off
+        loadPlayground(
+            app,
+            settings
+        )
+        app.buttons["Present PaymentSheet"].tap()
+        let payButton = app.buttons["Pay $50.99"]
+
+        // Select affirm
+        guard let affirm = scroll(collectionView: app.collectionViews.firstMatch, toFindCellWithId: "Amazon Pay") else {
```

Comment:
Rename to `amazonPay`.

Reply:
I think this should actually be Amazon Pay as I matched it in `StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift`. Otherwise the test will fail.

Reply:
Yeah the cell ID is correct but the `guard let affirm` is what I was referencing.

Reply:
oh gotcha, my bad haha